recipe/install.sh

```diff
@@ -0,0 +1,26 @@
+set -eu -o pipefail
+
+mkdir build && pushd build
+
+if [[ $(uname) == 'Darwin' ]] && [[ $PY3K == 1 ]]; then
+    export LDFLAGS="${LDFLAGS} -undefined dynamic_lookup"
+fi
+
+extra_opts="${@} "
+if [[ "${gpl_ok}" == 'True' ]]; then
+    extra_opts="-DREQUIRE_M4RI=ON ${extra_opts}"
+else
+    extra_opts="-DMIT=ON ${extra_opts}"
+fi
+
+# * ENABLE_TESTING: Testing requires lit which is not packaged yet:
+#   https://github.com/conda-forge/staged-recipes/issues/4630
+cmake \
+    -G "${CMAKE_GENERATOR}" \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+    -DENABLE_TESTING=OFF \
+    ${extra_opts} \
+    ..
+
+cmake --build . --target install -- -j${CPU_COUNT}
+popd && rm -r build
```

Comment:
The reason why I resorted to do full/redundant builds for each output is that the initial `cmake` already stored the Python paths in its config. That's obviously problematic since we want to swap this out for the `pycryptosat` output.
I just don't know if there is anything more sophisticated one can do with CMake here.
The current solution "ain't pretty but works" ¯\\_(ツ)\_/¯.